Platforms/QemuQ35Pkg/Library/EmuVariableFvbLib/EmuVariableFvbLib.inf

```diff
@@ -0,0 +1,43 @@
+## @file

```

Comment:
As mentioned offline, `gEfiMdeModulePkgTokenSpaceGuid.PcdEmuVariableNvModeEnable` provides built-in variable emulation now. Should remove FVB hacks related to variable service usage.